lib/pengine/pe_actions.c

```diff
@@ -965,7 +965,8 @@ pcmk__parse_on_fail(const pcmk_resource_t *rsc, const char *action_name,
      * failures that don't are probes and starts. The user can explicitly set
      * PCMK_META_ON_FAIL=PCMK_VALUE_FENCE to fence after start failures.
      */
-    if (pe__resource_is_remote_conn(rsc)
+    if (rsc->is_remote_node
```

Comment:
For sanity, `pcmk__parse_on_fail()` should NULL-check resource. (A NULL value would be a problem earlier than this point.)

Reply:
good point